sys/dev/pci/pci_user.c

```diff
@@ -997,7 +997,7 @@ pci_bar_mmap(device_t pcidev, struct pci_bar_mmap *pbm)
 		vm_object_deallocate(obj);
 		goto out;
 	}
-	pbm->pbm_map_base = (void *)addr;
+	pbm->pbm_map_base = (void *)(uintptr_t)addr;
```

Comment:
This seems wrong.  This is a user address, so both `addr` and `pbm_map_base` should be capabilities.  Possibly `pbm_map_base` needs `__kerncap`?